visual-js/visual-playwright/src/fixtures.ts

```diff
@@ -1,11 +1,23 @@
 import { TestFixture, TestInfo } from '@playwright/test';
 import { Page } from 'playwright-core';
 import { SauceVisualParams } from './types';
-import { sauceVisualCheck } from './playwright';
+import { sauceVisualCheck, sauceVisualResults } from './playwright';
+import { DiffStatus } from '@saucelabs/visual';
 
 export type SauceVisualFixtures = {
   sauceVisual: {
+    /**
+     * Takes a snapshot of the current page and uploads it to Sauce Labs for visual diffing.
+     * @param name
```

Comment:
can we add a descriptions for each parameters?

Reply:
Do `name` and `options` really need descriptions? 😅 Those seem pretty obvious.